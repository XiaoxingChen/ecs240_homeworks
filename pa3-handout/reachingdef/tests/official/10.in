c Irreducible loop
p 4 7 10 1 7
c bb1: v1 = v2;
b 1 1 2
e 1 2
e 1 6
c bb2: v2 = v1 + v3;
b 2 2 1 3
e 2 3
e 2 5
e 2 7
c bb3: v2 = c;
b 3 2
e 3 2
e 3 4
c bb4: v4 = v1 + v2 + v3;
b 4 4 1 2 3
e 4 3
c bb5: v2 = c;
b 5 2
e 5 4
c bb6: v3 = c;
b 6 3
e 6 4
c bb7: ;
b 7
