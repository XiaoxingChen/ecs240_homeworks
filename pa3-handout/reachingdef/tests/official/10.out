rdout 1 1
rdout 2 1 2 4 6
rdout 3 1 3 4 6
rdout 4 1 3 4 5 6
rdout 5 1 4 5 6
rdout 6 1 6
rdout 7 1 2 4 6
